articles/migrations/guides/liveid-api-deprecation.md

```diff
@@ -0,0 +1,135 @@
+---
+title: Migration to Azure AD Personal Accounts
+description: This article covers the Live Connect + SDK deprecation and how to update your Auth0 Microsoft Account Connection.
+toc: true
+contentType:
+  - how-to
+useCase:
+  - add-login
+  - migrate
+---
+
+# Microsoft Account Migration to Azure AD (personal accounts) + Microsoft Graph
+
+In October 2017, Microsoft announced the [deprecation of the Live Connect API and Live SDK](https://developer.microsoft.com/en-us/office/blogs/outlook-rest-api-v1-0-office-365-discovery-and-live-connect-api-deprecation). This is a Microsoft deprecation that will affect Auth0 users using the Microsoft social connection, and requires changes in the Auth0 configuration and potentially application code. 
+
+The change implies switching:
+
+- from the Live Connect API to the Azure Active Directory API to authenticate
+- from the Live SDK to Microsoft Graph to be able to get other resources including user profiles, contacts, files, etc
+
+You can decide if Auth0 uses Live Connect + Live SDK or Azure AD + Microsoft Graph using the 'Strategy Version' field in the Microsoft Account connection settings page. 
+
+![New Microsoft Connection Settings](/media/articles/connections/social/microsoft-account/microsoft-account-azureid.png)
+
+You need to switch to 'Azure AD (personal accounts)' to ensure your applications will keep working after Microsoft decommissions the API.
+
+**User Profile**
+
+The profile that can be accessed through Microsoft Graph provides different user profile information compared to the Live Connect and Live SDK user profile. In particular, some of the user profile fields that were previously available are not available anymore:
+ 
+
+***OIDC Profile***
+
+| Field  |  Live SDK |  Microsoft Graph  |
+|--------|---------------|------------------|
+|  picture | Returned a URL for the user picture | Auth0 will build a URL that will return a default picture for the user based on their initials. To get the actual image you need to [download it using Microsoft Graph](https://docs.microsoft.com/en-us/graph/api/profilephoto-get?view=graph-rest-1.0)|
+| locale | Returned a string in the format en_US | Not available. |
+
```

Comment:
I haven't tried this, but the linked Graph API doc says:
> Note This operation in version 1.0 supports only a user's work or school mailboxes and not personal mailboxes.
which means that it wouldn't be usable in this case.

Reply:
Yes, there's no support to get a picture afaik.